api/storage/api-docs/api-docs.json

```diff
@@ -414,6 +414,257 @@
                 }
             }
         },
+        "/api/v2/media": {
```

Comment:
If these are documented as decorators in the controller files (e.g., api/app/Http/Controllers/MediaController.php), I'm curious whether it's actually required to document them again here? Or perhaps this file was automatically generated as a consequence of the decorators?